!GameTexts/NewTexts.xml

```diff
@@ -744,7 +741,7 @@
 			<Text>Unit takes an additional 20 "bleed" damage [COLOR_POSITIVE_TEXT]at the start of its turn[ENDCOLOR].</Text>
 		</Row>
 		<Row Tag="TXT_KEY_PROMOTION_UNIT_POLYNESIA_KAPU_KUIALUA_3_ON_UNIT">
-			<Text>Massive Bleeding</Text>
+			<Text>Hemorrhaging</Text>
```

Comment:
Change also lua popup text.

Reply:
I don't know where to find the lua popup